sdk/appconfiguration/Azure.Data.AppConfiguration/samples/Azure.Data.AppConfiguration.Samples.Tests.csproj

```diff
@@ -2,6 +2,10 @@
   <PropertyGroup>
     <TargetFrameworks>$(RequiredTargetFrameworks)</TargetFrameworks>
   </PropertyGroup>
+  <!-- Disable StyleCop -->
```

Comment:
We should probably opt-out these test projects in the common props/targets file, no need to duplicate it for everyone.

Reply:
Style checking will eventually be enabled for tests just like the main csproj

Reply:
Assuming it doesn't currently work for any tests I still think disabling it in one common places is better then doing it in all projects.